### PR TITLE
Milestone 10: Add input to localStorage

### PR DESCRIPTION
**Pull request by yonas44**
- Add an object to store data from input fields.
- Store data in localStorage.
- Update the input fields using data from localStorage.

Thank you for reviewing our PR!

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -159,3 +159,43 @@ contactForm.addEventListener('submit', (event) => {
...
+
+if (localStorage.test === undefined) {
+  localStorage.setItem('test', JSON.stringify(sessionObj));
+}
+const testValue = localStorage.getItem('test');
+const testValueObj = JSON.parse(testValue);
+fullName.value = testValueObj.fullName;
+email.value = testValueObj.email;
+textArea.value = testValueObj.textArea;
+
```
Comment (gemmen29) [inline script.js:187]: - [x] [OPTIONAL] Please fix the following behavior:
   - When the user writes data in the inputs field, the data is preserved correctly in the local storage. However, when the user tries to add to the data imported from the local storage, the local storage objects get overwritten with the empty object and you lose the data preserved. 
 ### Hint: You can update the `sessionObj` object with data in the local storage like:
 ```
sessionObj.fullName = testValueObj.fullName;
sessionObj.email = testValueObj.email;
sessionObj.textArea = testValueObj.textArea;
```

**Review by gemmen29 — APPROVED** [review]
Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions
- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (yonas44) [reply script.js:187]: Okay, thanks for the feedback and suggestions!

Comment (manq2010) [reply script.js:187]: Thanks for the feedback!

**Pull request merged**